cf_remote/commands.py

```diff
@@ -556,8 +556,8 @@ def destroy(group_name=None):
 
 
 def list_platforms():
-    print("Available platforms:")
-    for key in sorted(cloud_data.aws_platforms.keys()):
+    print("Available platforms (image criteria):")
+    for key in sorted(cloud_data.aws_image_criteria.keys()):
```

Comment:
list platforms isn't quite descriptive anymore as we don't have an explicit list for AWS (I don't know anything (yet) about GCP).

new output:
```
cf-remote spawn --list-platforms
Available platforms (image criteria):
centos
debian
debian-9
macos
rhel
suse
ubuntu
ubuntu-16
windows
windows-2008
windows-2012
windows-2016
windows-2019
```

Reply:
Old output:

```
$ cf-remote spawn --list-platforms
Available platforms:
centos-5-x32
centos-5-x64
centos-6-x64
centos-7-x64
debian-10-x64
debian-11-arm64
debian-11-x64
debian-12-x64
debian-4-x32
debian-4-x64
debian-5-x32
debian-5-x64
debian-6-x32
debian-6-x64
debian-7-x32
debian-7-x64
debian-8-x64
debian-9-x64
rhel-5-x64
rhel-6-x64
rhel-7-x64
rhel-8-x64
rhel-9-x64
ubuntu-12-04-x32
ubuntu-12-04-x64
ubuntu-14-04-x32
ubuntu-14-04-x64
ubuntu-16-04-x64
ubuntu-18-04-x64
ubuntu-20-04-x64
ubuntu-22-04-arm64
ubuntu-22-04-x64
windows-2012-x64
windows-2016-x64
windows-2019-x64

```

Reply:
Yes, I tested several of the older platforms and pretty sure their AMIs are dead. But I will double-check those today.

Reply:
adjusted slightly for more information since we don't explicitly have a list anymore to describe the format of the "platform" argument.

```
Platform images are queried based on the platform name, version and architecture.
The form of platform specified is: <platform_name>[-<version>][-<architecture>]. e.g. debian, debian-12 or debian-12-x64
Ubuntu version can be just major (20) or major+minor (20-04)
Architecture can either be x64 or arm64

Available platforms:
centos
debian
debian-9
rhel
suse
ubuntu
ubuntu-16
windows
windows-2008
windows-2012
windows-2016
windows-2019
```